Dockerfile

```diff
@@ -14,7 +14,26 @@ RUN curl -L https://github.com/open-policy-agent/conftest/releases/download/v"${
     cd .. | rm -rf "bats-core-$BATS_VERSION" | rm -rf "v$BATS_VERSION.tar.gz" && \
     dnf -y --setopt=tsflags=nodocs install \
     jq \
-    skopeo 
+    skopeo \
+    wget \
+    unzip
+
+#To support JAVA
+RUN curl -L https://download.java.net/java/GA/jdk18.0.1.1/65ae32619e2f40f3a9af3af1851d6e19/2/GPL/openjdk-18.0.1.1_linux-x64_bin.tar.gz | tar -xz --no-same-owner -C /home/
```

Comment:
Tip: combine the separate `RUN` instructions into a single one to prevent creating a [separate layer for each instruction](https://testdriven.io/tips/a02d1c1b-81a8-49eb-938c-00cb29eb7896/) in the image.

Reply:
Yes, thank you! Let me combine them